sdk/messaging/azservicebus/internal/amqp_test_utils.go

```diff
@@ -8,6 +8,7 @@ import (
 	"fmt"
 
 	"github.com/Azure/azure-sdk-for-go/sdk/internal/log"
+	"github.com/Azure/azure-sdk-for-go/sdk/messaging/azservicebus/internal/amqpwrap"
```

Comment:
Unrelated to this PR.

Is the contents of this file used only for tests?  If so, shouldn't the name be `amqp_utils_test.go` to limit it to tests?

Reply:
I wanted that too - it's re-used by tests in the azservicebus package and sub-package _test.go files aren't available in that case.